great_expectations/expectations/metrics/column_map_metrics/column_values_string_integers_monotonically_increasing.py

```diff
@@ -0,0 +1,77 @@
+import pandas as pd
+import numpy as np
+from typing import Optional
+
+from great_expectations.core import ExpectationConfiguration
+from great_expectations.execution_engine import (
+    ExecutionEngine,
+    PandasExecutionEngine,
+    SparkDFExecutionEngine,
+)
+from great_expectations.execution_engine.execution_engine import MetricDomainTypes
+from great_expectations.execution_engine.sqlalchemy_execution_engine import (
+    SqlAlchemyExecutionEngine,
+)
+from great_expectations.expectations.metrics.import_manager import F, sa
+from great_expectations.expectations.metrics.map_metric_provider import (
+    ColumnMapMetricProvider,
+    column_condition_partial,
+    column_function_partial,
+)
+from great_expectations.validator.metric_configuration import MetricConfiguration
+
+
+class ColumnValuesStringIntegersMonotonicallyIncreasing(ColumnMapMetricProvider):
+    function_metric_name = "column_values.string_integers.monotonically_increasing"
+    function_value_keys = tuple()
+
+    @column_function_partial(engine=PandasExecutionEngine)
+    def _pandas_function(self, data, _metrics, **kwargs):
+
+        try:
+            temp_column = [int(x) for x in data if x.isdigit()]
+        except AttributeError:
+            raise AttributeError(
+                "Column must be a string-type capable of being cast to int."
+            )
+            return False
+
+        series_diff = np.diff(temp_column)
+
+        return series_diff >= 0
+
+
+    @column_function_partial(engine=SparkDFExecutionEngine)
+    def _spark_function(self, data, _metrics, **kwargs):
+
+        temp_column = data.select(column).rdd.flatMap(lambda x: x).collect()
+        try:
+            temp_column = [int(x) for x in data if x.isdigit()]
+        except AttributeError:
+            raise AttributeError(
+                "Column must be a string-type capable of being cast to int."
+            )
+            return False
+
+        series_diff = np.diff(temp_column)
```

Comment:
This is where you have heard us discussing that the Spark implementation does not actually perform a calculation on data. Instead you need to return the Spark logic that will perform the computation at a later time.